src/emu/debugger/src/imgui_debugger.cpp

```diff
@@ -495,6 +497,17 @@ namespace eka2l1 {
         ImGui::Separator();
     }
 
+    struct SLocaleLanguage {
```

Comment:
Can you define this struct somewhere like in epoc/utils? We so can share this struct without having to define it multiple times.

Reply:
Moved to epoc/utils/locale.h, and I copied definition of enum TLanguage to there too since including e32lang.h causes compiling errors, is that OK?

Reply:
That's ok, thanks